src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const convertedString = sourceString
+    .split(';')
+    .filter((style) => style.trim() !== '')
+    .reduce((prev, element) => {
+      const splittedStyle = element.split(':').map((el) => el.trim());
```

Comment:
```suggestion
      const [property, value] = element.split(':').map((el) => el.trim());
```
You can use descturcting